src/shared/seo.tsx

```diff
@@ -27,8 +27,9 @@ function SEO({
         { property: `og:description`, content: description },
         { property: `og:image`, content: metaImage },
```

Comment:
This is still using tallyho.org 👀

Reply:
Looks like that's at https://github.com/tahowallet/taho.xyz/blob/10299c0475c599ca7b1db6a4e8c2d7fcebde96bd/src/shared/seo.tsx#L9